scriptmodules/libretrocores/lr-mgba.sh

```diff
@@ -40,11 +40,12 @@ function install_lr-mgba() {
 }
 
 function configure_lr-mgba() {
-    mkRomDir "gba"
-    ensureSystemretroconfig "gba"
-
     local def=1
     isPlatform "armv6" && def=0
-    addEmulator $def "$md_id" "gba" "$md_inst/mgba_libretro.so"
-    addSystem "gba"
+    for system in gb gbc gba; do
```

Comment:
You should ensure the ```system``` variable is local scope

Reply:
oh forgot that, thanks for spotting.